### PR TITLE
Add link next to LP balance

### PR DESCRIPTION
**Pull request by alexgeorgousis**

### CODE REVIEW

File: src/components/minting/mintingFormControl.tsx [hunk 1]
```diff
@@ -70,6 +71,14 @@ const MintingFormControl: FC = () => {
               Balance:{' '}
               <strong>
                 {lpBalance ? formatNumber.token(lpBalance) : '-'} LP
+                <Link
+                  href='http://lp.mist.alchemist.wtf'
```
Comment (joehawary) [inline src/components/minting/mintingFormControl.tsx:75]: We have some config variables for links, I believe you should be able to pick it up from the footer.

This is due to us using different links on testnets :)

**Review by joehawary — APPROVED** [review]
Awesome work @alexgeorgousis, Nice little addition there, looks the part!

File: src/components/minting/mintingFormControl.tsx (hunk 1 above)
Comment (alexgeorgousis) [reply src/components/minting/mintingFormControl.tsx:75]: Oh cool, I can add this to the list and get it from there :)

**Comment by joehawary** [comment]
Looks perfect! Great job! 🥇

**Pull request merged**